code/app/build.gradle.kts

```diff
@@ -42,12 +42,15 @@ dependencies {
     implementation(libs.activity)
     implementation(libs.constraintlayout)
     implementation(libs.espresso.intents)
+    implementation("com.squareup.picasso:picasso:2.71828")
```

Comment:
do we need this import?

Reply:
It helps with the pictures so have in case we plan to use it

Reply:
if we are not using this now, then we don't need it, we can always add it later in case it is used